src/service/frameworkTermService.js

```diff
@@ -257,30 +263,89 @@ function frameworkTermRetire (req, response) {
   async.waterfall([
 
     function (CBW) {
-      logger.debug({ msg: 'Request to get Framework Terms', additionalInfo: { data } }, req)
-      ekStepUtil.frameworkTermRetire(data.category, req.headers, data.queryParams, function (err, res) {
-        if (err || res.responseCode !== responseCode.SUCCESS) {
-          rspObj.responseCode = res && res.responseCode ? res.responseCode : responseCode.SERVER_ERROR
-          logger.error({
-            msg: 'Error while fetching framework terms from ekstep',
-            err: {
-              err,
-              responseCode: rspObj.responseCode
-            },
-            additionalInfo: {data}
-          }, req)
-          var httpStatus = res && res.statusCode >= 100 && res.statusCode < 600 ? res.statusCode : 500
-          rspObj.result = res && res.result ? res.result : {}
-          rspObj = utilsService.getErrorResponse(rspObj, res)
+      var ekStepReqData = {
+        request: {
+          filters: {
+            identifier: data.request.contentIds,
+            status: []
+          }
+        }
+      }
+    },
+
+    function (res, CBW) {
+      var createdByOfContents = _.uniq(_.pluck(res.result.term, 'createdBy'))
```

Comment:
@anilkumarkammalapalli  Please check from which this  createdByContent append.
@karthik-tarento  What should be used here that created only can delete this?